include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -373,6 +371,8 @@ SECTIONS
         KERNEL_INPUT_SECTION(".noinit.*")
 	*(".kernel_noinit.*")
 
+#include <linker/priv_stacks-noinit.ld>
+
```

Comment:
maybe this needs to be at the very very very very end.  past the soc area too.

Reply:
@agross-linaro please have a look at #10542